features/steps/thens/alignment.py

```diff
@@ -63,8 +80,8 @@ def step_impl(context, inst, ifc_rep_criteria, ifc_layout_criteria):
                                                 severity=OutcomeSeverity.ERROR)
 
             case ("IfcGradientCurve", "presence of IfcAlignmentVertical"):
```

Comment:
This will then match the following
```python
match (ifc_rep_criteria, existance, entities):
  case ("IfcGradientCurve", "presence", "IfcAlignmentVertical")
```